apps/main/src/loan/components/PageLoanCreate/Page.tsx

```diff
@@ -94,49 +94,41 @@ const Page = () => {
   )
 
   useEffect(() => {
-    if (pageLoaded && curve) {
-      if (llamma) {
-        resetUserDetailsState(llamma)
-        fetchInitial(curve, isLeverage, llamma)
-        void fetchLoanDetails(curve, llamma)
-        setLoaded(true)
-      } else if (collateralDatasMapper) {
```

Comment:
I accidentally removed the redirect if market does not exist / could not be found. Do I really need to bring that back? We don't do this for lending markets either fwiw...

I personally dislike redirects

Reply:
Either we add redirects or we show a proper 404 error. Currently it just loads forever